### ISSUE TITLE
License Clarification

### ISSUE THREAD

**Issue by QuantumPlayDev** [huawei-noah/HEBO#101]
Hiya, this is cool and thanks so much for sharing. Just wanted to clarify license?

**Comment by AntGro** [comment]
Hi, you would like to know the license of which of the project we list in the README?

**Comment by QuantumPlayDev** [comment]
> Hi, you would like to know the license of which of the project we list in the README?

Thank you very much for your quick reply. I just checked the readme but unfortunately I didn't see the license specified there. Is there any chance to add it to the license section of the repository? Very sorry to take up your time with this, just want to make sure I comply with your licensing!

Have a great day.

**Comment by QuantumPlayDev** [comment]
It may just be AIRBO, PMDB, SIMMER that appear to missing the licenses. Just realized the repo is utilizing different licenses for the various components.

**Comment by AntGro** [comment]
AIRBO: @yanglin-jason 
PMDB: @kygguo 
SIMMER: @aivarsoo

**Comment by aivarsoo** [comment]
> AIRBO: @yanglin-jason PMDB: @kygguo SIMMER: @aivarsoo

Hi, I missed the license for SIMMER, I assumed there was a license for the whole repo. Which license do you think is appropriate for SIMMER? I am happy with any, but I don't think it's my decision :)

**Comment by QuantumPlayDev** [comment]
Of course you can :-) MIT seems classy

**Comment by AntGro** [comment]
> > AIRBO: @yanglin-jason PMDB: @kygguo SIMMER: @aivarsoo
> 
> Hi, I missed the license for SIMMER, I assumed there was a license for the whole repo. Which license do you think is appropriate for SIMMER? I am happy with any, but I don't think it's my decision :)

Indeed, there should be one license file per project, if the code does not include third party components that were not released under MIT license, then MIT license can ve used.

**Comment by aivarsoo** [comment]
See https://github.com/huawei-noah/HEBO/pull/102 for Simmer license PR. I am not sure what language you should use to reflect that the code is based on safety starter agents and safety gym. Please double check

**Comment by AntGro** [comment]
Could you include the list of third party codes Simmer code is built on? Like including a [THIRD PARTY OPEN SOURCE SOFTWARE NOTICE.txt file](https://github.com/huawei-noah/HEBO/blob/master/MCBO/THIRD%20PARTY%20OPEN%20SOURCE%20SOFTWARE%20NOTICE.txt)

**Comment by aivarsoo** [comment]
sorry for the delay! updated PR
